lib/app.js

```diff
@@ -23,7 +23,7 @@ var defaultSettings = {
 
 function getPrompt(argv, schema, onComplete, index, results) {
   if (!Array.isArray(schema)) {
-    console.log(chalk.red('An error occurred'));
+    console.log(chalk.red('getPrompt schema must be an Array.'));
```

Comment:
My understanding is that it is a debug message. Our users can never see this even for invalid inputs. So, I just changed that to something we can use for debugging. Do you want me to revert it ?